modules/099-ceph-csi/docs/USAGE.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "The ceph-csi module: usage"
+---
+
+## An example of the `CephCSI` configuration
```

Comment:
You changed the kind name to `CephCSIDriver` but forgot to change examples.

Reply:
Fixed.